docs-src/concepts/what-is-a-schedule.md

```diff
@@ -192,3 +192,5 @@ A Workflow started by a Schedule can obtain the details of the failure of the mo
 Internally, a Schedule is implemented as a Workflow.
 If you're using Advanced Visibility (Elasticsearch), these Workflow Executions are hidden from normal views.
 If you're using Standard Visibility, they are visible, though there's no need to interact with them directly.
+
+For triggering a Workflow Execution at a specific one-time future point rather than on a recurring schedule, the [Start Delay](/concepts/what-is-a-delay-start-workflow-execution) option should be used instead of a Schedule.
```

Comment:
I think this would make more sense as an info box / aside higher up, above "These two kinds have multiple representations"

Reply:
Would Schedules inability to run once, be a limitation of a Schedule, and thus, be under the limitation section?

Reply:
A Schedule can run once. Folks are doing that now because they don't know about start delay. I think more of them will find out it exists if this note is higher up, in the schedule spec section.

Reply:
I want to make sure we are messaging the schedules and startdelay relationship properly, having some internal convos about this.